src/containers/apps/CreateNewApp.tsx

```diff
@@ -101,13 +103,22 @@ class CreateNewApp extends Component<
                             </Tooltip>
                         </Row>
                     </Col>
-                    <Col lg={{ span: 12 }}>
-                        <div style={{ textAlign: 'center' }}>
-                            <p>Or Select From</p>
-                            <Link to="/apps/oneclick/" className="ant-btn">
-                                One-Click Apps/Databases
-                            </Link>
-                        </div>
+                    {self.props.isMobile ? (
```

Comment:
Was this needed for the migration? If not, I suggest keeping this PR to minimum that's needed for the migration and post these changes in a subsequent PR.